src/main/java/org/softc/armoryexpansion/common/integration/aelib/config/ArmorMaterialConfigOptions.java

```diff
@@ -0,0 +1,78 @@
+package org.softc.armoryexpansion.common.integration.aelib.config;
+
+import org.softc.armoryexpansion.common.integration.aelib.plugins.general.material.IBasicMaterial;
+
+public class ArmorMaterialConfigOptions {
+    protected String name;
+    protected boolean material;
+    private boolean fluid;
+    protected boolean traits;
+
+    protected ArmorConfigOptions armorOptions;
+
+    ArmorMaterialConfigOptions(IBasicMaterial iBasicMaterial) {
+        this.name = iBasicMaterial.getIdentifier();
+        this.material = true;
+        this.fluid = iBasicMaterial.isCastable();
+        this.traits = true;
+
+        this.armorOptions = new ArmorConfigOptions(iBasicMaterial.isArmorMaterial());
+    }
+
+    protected ArmorMaterialConfigOptions(String name, boolean material, boolean traits, boolean armorOptions) {
+        this.name = name;
+        this.material = material;
+        this.traits = traits;
+        this.armorOptions = new ArmorConfigOptions(armorOptions);
+    }
+
+    String getName() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896609)